client/src/main/scala/org/apache/celeborn/client/commit/ReducePartitionCommitHandler.scala

```diff
@@ -242,6 +246,7 @@ class ReducePartitionCommitHandler(
 
   override def registerShuffle(shuffleId: Int, numMappers: Int): Unit = {
     super.registerShuffle(shuffleId, numMappers)
+    getReducerFileGroupRequest.put(shuffleId, new util.HashSet[RpcCallContext]())
```

Comment:
Do init for registered shuffle here to reduce a `contains` check when handle `GetReducerFileGroupRequest`